examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -528,10 +528,11 @@ class SubscribeEvent : public ReportCommand
         ReportCommand("subscribe-event", credsIssuerConfig),
         mClusterIds(1, clusterId), mEventIds(1, eventId)
     {
-        AddArgument("attr-name", eventName);
-        AddArgument("min-interval", 0, UINT16_MAX, &mMinInterval);
-        AddArgument("max-interval", 0, UINT16_MAX, &mMaxInterval);
-        AddArgument("keepSubscriptions", 0, 1, &mKeepSubscriptions);
+        AddArgument("attr-name", eventName, 0, "Event name.");
```

Comment:
While we are here:
```suggestion
        AddArgument("event-name", eventName, 0, "Event name.");
```

Reply:
Yup, I thought it was strange it was called attr-name here.  Will change it.